src/components/categories/CategoryList.js

```diff
@@ -29,19 +29,34 @@ export default (props) => {
     <>
       <article>
           {categories.map((c) => (
-            <ListGroup key={c.id}>
-              <ListGroup.Item className="mb-2">
-              <Row>
-                <Col className="mt-2">
-              <Category category={c} key={c.id} />
+            <Row className="align-items-center">
```

Comment:
I'm seeing a react warning that "index.js:1 Warning: Each child in a list should have a unique "key" prop." We should add a key here

Reply:
got it whoops that should be updated now